<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addProductWidget</class>
 <widget class="QWidget" name="addProductWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrc.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Название товара</string>
   </property>
  </widget>
  <widget class="QLabel" name="keyWordLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>115</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Ключевое слово</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Описание товара</string>
   </property>
  </widget>
  <widget class="QLabel" name="purchasePriceLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>285</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена закупки, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="sellingPriceLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>350</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена продажи, руб</string>
   </property>
  </widget>
  <widget class="QLabel" name="taxLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>415</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Налог, %</string>
   </property>
  </widget>
  <widget class="QLabel" name="profitabilityLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>480</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Рентабельность</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>610</y>
     <width>195</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>725</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="keyWordEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>115</y>
     <width>725</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="purchasePriceEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>285</y>
     <width>725</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sellingPriceEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>725</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>415</y>
     <width>725</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>180</y>
     <width>725</width>
     <height>90</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="productPic">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>50</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="profitabilityValueLabel">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>480</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="shelfLifeLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>545</y>
     <width>175</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок Годности, дн</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="shelfLifeEdit">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>545</y>
     <width>725</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addPicButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>275</y>
     <width>195</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>610</y>
     <width>175</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
